docs/console.html

```diff
@@ -0,0 +1,804 @@
+                    <figure>
+                        <pre class="prettyprint"><code class="language-javascript">
+console.assert(list.childNodes.length < 500, "Node count is > 500");              
+</code></pre>
+                    </figure>
+                </figure>
+                <figure>
+                    <img src="console-files/assert-failed.png" alt="Assertion failed" />
+                    <figcaption>
+                        How an assertion failure displays in the console.
+                    </figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="filtering-console-output">Filtering console output</h3>
+            <p>
+                You can filter console output by its severity level by selecting one of the filter options.
+                Filters are enabled under the filter funnel icon located in the upper-left corner of the console panel.
+                The following filter options are available:
+            </p>
+
+            <dl>
+                <dt>All</dt>
+                <dd>Shows all console output.</dd>
+
+                <dt>Errors</dt>
+                <dd>Only show output from <code>console.error()</code></dd>
+                
+                <dt>Warnings</dt>
+                <dd>Only show output from <code>console.warn()</code></dd>
+                
+                <dt>Logs</dt>
+                <dd>Only show output from <code>console.log()</code>, <code>console.info()</code> and <code>console.debug()</code>.</dd>
+                
+                <dt>Debug</dt>
+                <dd>Only show output from <code>console.timeEnd()</code> and other console output.</dd>
+            </dl>
+
+            <figure>
+                <img src="console-files/filter-errors.png" alt="Filtering errors" />
+                <figcaption>
+                    Filter enabled for showing only errors.
+                </figcaption>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="grouping-output">Grouping Output</h3>
+            <p>
+                You can group related output together with the group commands.
+                The <code><a href="console-api#consolegroupobject-object">group</a></code> command takes a singe string parameter to set the name of the group.
+                One this is called, the console will begin to group all subsequent output together.
+                To end the grouping you only need to call <code><a href="console-api#consolegroupend">groupEnd()</a></code>.
+            </p>
+
+            <figure>
+                <figcaption>Simple grouping example</figcaption>
+                <figure>
+                    <figcaption>Example code:</figcaption>
+                    <pre class="prettyprint"><code class="language-javascript">
```

Comment:
No clue. Dragged the content out into `<pre class="prettyprint">` and it is still just green. So not an issue with the code block.

Could be an issue with the preview mode only... Things do at least look right in production. Could be the script to actually setup the pre block with the right elements to style isn't executing in _preview.